airbyte-scheduler/src/main/java/io/airbyte/scheduler/persistence/DefaultJobPersistence.java

```diff
@@ -526,9 +539,29 @@ private void importTable(DSLContext ctx, final String schema, final DatabaseSche
         // LOGGER.debug(insertStep.toString());
         ctx.batch(insertStep).execute();
       }
+      final Optional<Field<?>> idColumn = columns.stream().filter(f -> f.getName().equals("id")).findFirst();
+      if (idColumn.isPresent())
+        resetIdentityColumn(ctx, schema, tableType);
+    }
+  }
+
+  private static void resetIdentityColumn(final DSLContext ctx, final String schema, final DatabaseSchema tableType) {
```

Comment:
is this resetting what the next primary id will be? if not i'm not sure what this is doing. can you add a comment explaining?

Reply:
yes exactly it reset the next value used by auto-increment